build-logic-commons/module-identity/src/main/kotlin/gradlebuild/identity/extension/ReleasedVersionsDetails.kt

```diff
@@ -31,10 +31,12 @@ class ReleasedVersionsDetails(currentBaseVersion: GradleVersion, releasedVersion
     val allTestedVersions: List<GradleVersion>
 
     val mainTestedVersions: List<GradleVersion>
+    val lowestInterestingVersion: GradleVersion
+    val lowestTestedVersion :GradleVersion
 
     init {
-        val lowestInterestingVersion = GradleVersion.version("0.8")
-        val lowestTestedVersion = GradleVersion.version("3.0")
+        lowestTestedVersion = GradleVersion.version("3.0")
+        lowestInterestingVersion = GradleVersion.version("0.8")
```

Comment:
I don't fully understand why we need this variable. Isn't this just a leftover from the past?

Reply:
We at least use it to generate `allPreviousVersions` so I'd keep it for now.